### PR TITLE
Object exercises

### PR DESCRIPTION
**Pull request by fadime94**

### CODE REVIEW

File: object-exercises/object-exercises (2).js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+//"Your TP will only last 2 days, buy more!"
+//"Your TP will last 26 days, no need to panic!"
+
+function tpChecker(obj){
+  const quarantineDays = 14;
+  const totalSheet = 500;
+  const dailySheet= 57;
+  const people = obj.people;
+  const tpAvailable = obj.tp;
```
Comment (Halit-Batur) [inline object-exercises/object-exercises (2).js:9]: really good variable names and avoiding useage of magic numbers !

**Comment by Halit-Batur** [comment]
@fadime94 good work ! Could you create a dir inside the object-exercise with your name and then put the .js file in it

File: object-exercises/object-exercises (2).js (hunk 1 above)
Comment (fadime94) [reply object-exercises/object-exercises (2).js]: thanks! :)

**Comment by Halit-Batur** [comment]
@fadime94 good work !

**Pull request merged**